README.md

```diff
@@ -40,36 +40,21 @@ composer require amphp/http-server
 ```php
 <?php
 
-use Amp\Http\Server\RequestHandler\CallableRequestHandler;
-use Amp\Http\Server\HttpServer;
 use Amp\Http\Server\Request;
 use Amp\Http\Server\Response;
-use Amp\Http\Status;
-use Amp\Socket\Server;
-use Psr\Log\NullLogger;
-
-// Run this script, then visit http://localhost:1337/ in your browser.
-
-Amp\Loop::run(function () {
-    $sockets = [
-        Server::listen("0.0.0.0:1337"),
-        Server::listen("[::]:1337"),
-    ];
-
-    $server = new HttpServer($sockets, new CallableRequestHandler(function (Request $request) {
-        return new Response(Status::OK, [
-            "content-type" => "text/plain; charset=utf-8"
-        ], "Hello, World!");
-    }), new NullLogger);
-
-    yield $server->start();
-
-    // Stop the server gracefully when SIGINT is received.
-    // This is technically optional, but it is best to call Server::stop().
-    Amp\Loop::onSignal(SIGINT, function (string $watcherId) use ($server) {
-        Amp\Loop::cancel($watcherId);
-        yield $server->stop();
-    });
+use function Amp\Http\Server\handleFunc;
+use function Amp\Http\Server\html;
+use function Amp\Http\Server\listenAndServe;
+
+// Run this script, then visit http://localhost:8000/ in your browser.
+
+function helloWorld(Request $request): Response
+{
+    return html('Hello World!');
+}
+
+Amp\Loop::run(static function () {
+    yield listenAndServe('0.0.0.0:8000', handleFunc('helloWorld'));
```

Comment:
Misses the shutdown now.

Reply:
Oops. Sorry!

Reply:
I just need to do `return yield listenAndServe()` right?

Reply:
Not quite sure if I get what you mean.